tests/sentry/snuba/metrics/test_metrics_layer/test_metrics_enhanced_performance.py

```diff
@@ -1432,7 +1438,7 @@ def test_team_key_transactions_my_teams(self):
                 name=TransactionMRI.DURATION.value,
                 tags={"transaction": transaction},
                 value=value,
-                minutes_before_now=minutes,
+                minutes_before_now=-minutes,
```

Comment:
I'm not sure why this worked before, if this was the right behavior, but I think the logic of it makes sense (positive minutes puts it before the interval queried)